micrometer-core/src/test/java/io/micrometer/core/instrument/binder/jvm/JvmGcMetricsTest.java

```diff
@@ -61,6 +73,49 @@ void metersAreBound() {
         assertThat(registry.find("jvm.gc.memory.promoted").counter()).isNotNull();
     }
 
+    @Test
+    @Disabled("Garbage collection can happen before JvmGcMetrics are registered, making our metrics not match overall counts/timings")
```

Comment:
While the test worked consistently for me on IntelliJ with ZGC and Shenandoah, it failed on the command line because of GC happening before JvmGcMetrics is registered, so the information in the GarbageCollectorMXBeans and our metrics do not add up. This is the issue described in #2561.